### PR TITLE
Add Sergey Rubanov as a reviewer for Russian translations + add some proposal description translations

### PR DESCRIPTION
**Pull request by chicoxyzzy**
I will add missing translations later or will try to find someone from Russian-speaking community to help with them.

### CODE REVIEW

File: ru/stage3.json [hunk 1]
```diff
@@ -41,10 +41,22 @@
...
-    "description": "Proposal for an Object.hasOwn() method to make Object.prototype.hasOwnProperty() more accessible."
+    "description": "Предложение по добавлению метода Object.hasOwn(), который делает Object.prototype.hasOwnProperty() более доступным."
   },
   "proposal-resizablearraybuffer": {
     "title": "In-Place Resizable and Growable ArrayBuffers",
     "description": null
+  },
+  "proposal-array-find-from-last": {
+    "title": "Array.prototype.findLast and Array.prototype.findLastIndex",
+    "description": "Методы по поиску соответствия с конца масси для элементов и их индексов."
```
Comment (mkrl) [inline ru/stage3.json]:
```suggestion
    "description": "Методы по поиску соответствия с конца массива для элементов и их индексов."
```
Just a small drive-by typo fixup

Comment (chicoxyzzy) [reply ru/stage3.json]: thanks!

**Review by ryzokuken — APPROVED** [review]
Everything LGTM except the translations, which I trust should be correct. Thanks!

**Review by codehag — APPROVED** [review]
great! thanks for doing this!
